editor/project_manager.cpp

```diff
@@ -843,7 +843,7 @@ void ProjectManager::shortcut_input(const Ref<InputEvent> &p_ev) {
 			} break;
 			case Key::F: {
 				if (k->is_command_or_control_pressed()) {
-					search_box->grab_focus();
+					this->search_box->grab_focus_edit();
```

Comment:
What's the `this` for?

Reply:
Maybe it was added by mistake, because i have copied the changes while working on 4.2.1 stable branch.